packages/helm-charts/attestation-bot/templates/secret.yaml

```diff
@@ -12,4 +12,5 @@ type: Opaque
 data:
   mnemonic: {{ .Values.mnemonic | b64enc }}
   twilioAccountSid: {{ .Values.twilio.accountSid | b64enc }}
+  twilioAddressSid: {{ .Values.twilio.addressSid | b64enc }}
```

Comment:
is this actually used by the bot any more?

Reply:
https://github.com/celo-org/celo-monorepo/blob/master/packages/env-tests/src/tests/attestation.ts#L80